app/src/main/res/values/strings.xml

```diff
@@ -311,6 +311,7 @@
   <string name="profile_edit_delete_dialog_negative">Cancel</string>
   <string name="profile_edit_allow_download_heading">Allow Download Access</string>
   <string name="profile_edit_allow_download_sub">User is able to download and delete content without Administrator password</string>
+  <string name="profile_edit_activity_title">Edit Profile</string>
```

Comment:
You will need to create similar string in other `strings.xml` file also like arabic language.

Reply:
Okay , working on it